src/Integration.UnitTests/LocalServices/CredentialStoreTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using FluentAssertions;
+using Microsoft.Alm.Authentication;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.LocalServices
+{
+    [TestClass]
+    public class CredentialStoreTests
+    {
+        [TestMethod]
+        public void DeleteCreds()
+        {
+            // Arrange
+            var mockCredentialStore = new Mock<ICredentialStore>();
```

Comment:
@duncanp-sonar how do you feel about TestInitialize? 
I think the Arrange blocks should move there, as they're a repetition in every test.

Reply:
My main goal is that unit tests should be completely independent of each other i.e. changing one test shouldn't impact another test. Both `TestInitialize` and sharing data between tests go against this, so in general I'm not in favour of `TestInitialize` as it tends to be misused. It's only really useful when all of the tests have exactly the same set of prerequisites which often isn't the case. There are lots of examples in the SLVS tests of `TestInitialize` being misused in this way. I tend to write tests as standalone first (i.e. without a `TestInitailize`), and then don't generally consider adding one later.

Unusually, this set of tests is currently simple enough to use TestInitialize, so I'll change them.

Constants - it depends. Here, we don't really care what the Uri is so making a shared const is ok. Generally, I'd want to vary the test data within each test so I wouldn't use constants.